spotify

```diff
@@ -188,7 +189,10 @@ while [ $# -gt 0 ]; do
         "vol"    )
             vol=`osascript -e 'tell application "Spotify" to sound volume as integer'`;
             text=$2;
-            if [ "$2" = "up" ]; then
+            if [ $2 = "show" ]; then
```

Comment:
Variable $2 should be double quoted :)
See https://github.com/koalaman/shellcheck/wiki/SC2086